### ISSUE TITLE
adonis na handling

### ISSUE THREAD

**Issue by krmaas** [vegandevs/vegan#302]
feature request

Can na.action (a la capscale) be added to adonis (or adonis2)? I thought that rm.na=TRUE did the same thing but apparently not.

**Comment by jarioksa** [comment]
True: missing value handling is not implemented in `adonis`. It seems that implementing this feature needs more than one iteration, though.

The argument would be `na.action`, though, similarly as in `dbrda` (and `capscale`).
